cerberus/tests/conftest.py

```diff
@@ -1,12 +1,49 @@
 # -*- coding: utf-8 -*-
 
 from copy import deepcopy
+from pkg_resources import Distribution, DistributionNotFound
 
 import pytest
 
 from cerberus import Validator
 
 
+def reload_module(name):
```

Comment:
though this is no runtime code, i'd still prefer to rely on `cerberus.platform.PYTHON_VERSION` instead of a nested try-except-block.

Reply:
sure, that might be nicer! using `reload` across multiple python versions is a pain.